README.md

```diff
@@ -1,2 +1,61 @@
-# Project
-DAY2 BLOCK WEEK1
+# Hello Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+
+One page website
+
+
+## Built With
+
+- html 
+- css
+- visual studio.NET
+- github
+
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Prerequisites
+VS code
+linters
+browser
+
+### Setup
+copy the url to your local https://github.com/jaamanu/Project.git
+
+## JAMES ASIBEY MANU
+
+👤 **Author1**
+
+- GitHub: [https://github.com/jaamanu](https://github.com/jaamanu)
+- Twitter: [@JamesAsibeyManu](https://twitter.com/JamesAsibeyManu)
+- LinkedIn: (https://www.linkedin.com/in/james-asibey-manu-a41b1430/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- Please, kindly add the missing licensed content as the link is broken to make your readme more professional